mail-filter/rspamd/rspamd-2.1.ebuild

```diff
@@ -34,6 +34,7 @@ RDEPEND="
 	cpu_flags_x86_ssse3? ( dev-libs/hyperscan )
 	jemalloc? ( dev-libs/jemalloc )
 	jit? ( dev-lang/luajit:2 )
+	!jit? ( dev-lang/lua:= )
```

Comment:
This essentially doesn't do anything, I think you wanted to do `:*` here.
Same applies to other rspamd versions.

Reply:
Well, I saw masked slots in `dev-lang/lua` and thought that maybe it would be necessary to rebuild `rspamd` once they will be unmasked. Thanks, it is changed to `:*`.